python/pyspark/pandas/frame.py

```diff
@@ -10260,14 +10262,16 @@ def rank(self, method: str = "average", ascending: bool = True) -> "DataFrame":
             * dense: like 'min', but rank always increases by 1 between groups
         ascending : boolean, default True
             False for ranks by high (1) to low (N)
+        numeric_only : bool, optional
+            Rank only numeric columns if set to True.
```

Comment:
nit: in pandas documents, it's mentioned as "For DataFrame objects, rank only numeric columns if set to True.".

Can we match the description as same as pandas ??

Reply:
Cool modified.